doc/mssqlserver/doc.md

```diff
@@ -0,0 +1,145 @@
+Galleon Layers
+=========
+
+* `mssqlserver-datasource`: Provision the `MSSQLServerDS` non xa datasource. Depends on `mssqlserver-jre8-driver` layer.
+* `mssqlserver-jre8-driver`: Provision the `mssqlserver-jre8` driver.
+
+Configuration
+========
+
+The following set of environment variables and corresponding Java system properties can be used to configure the datasource.
+
+Required configuration
+==============
+
+* `MSSQLSERVER_DATABASE`
+
+  * Description: Defines the database name to be used in the datasource’s `connection-url` property.
+  * Default Value: `NONE`
+  * Required: True if `MSSQLSERVER_URL` is not set.
+  * System Property: `org.jboss.eap.datasources.mssqlserver.database`
+
+* `MSSQLSERVER_HOST` (or `MSSQLSERVER_SERVICE_HOST`)
+
+  * Description: Defines the database server’s host name or IP address to be used in the datasource’s `connection-url` property.
+  * Default Value: `NONE`
+  * Required: True if `MSSQLSERVER_URL` is not set.
+  * System Property: `org.jboss.eap.datasources.mssqlserver.host`
+
+* `MSSQLSERVER_PORT` (or `MSSQLSERVER_SERVICE_PORT`)
+
+  * Description: Defines the database server’s port to be used in the datasource’s `connection-url` property. 
+  * Default Value: `NONE`
```

Comment:
is there a default standard port for all these databases?
E.g. PostgreSQL default port is `5432`

Reply:
Actually both host (locahost) and port could have a default value.